### ISSUE TITLE
Custom tag export does not work

### ISSUE THREAD

**Issue by MaudVandevelde** [dynatrace-oss/terraform-provider-dynatrace#190]
**Describe the bug**
It seems that the new added custom_tags do not work.
When I launch an export I do not have the custom tag 

unknown resource `dynatrace_custom_tags`

**To Reproduce**
1. Launch .\terraform-provider-dynatrace_v1.18.1.exe -export dynatrace_custom_tags
2. Error displayed: unknown resource `dynatrace_custom_tags`

**Expected behavior**
List of custom tags

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello @MaudVandevelde ,

You're right the resource `dynatrace_custom_tags` is not getting exported at the moment.
But there's a reason for that. Contrary to other endpoints of the Dynatrace REST API you're able to specify an `entitySelector` here. Essentially this means that a resource export would have to produce a `.tf` file for every single entity available within Dynatrace - every Service, every Host, every Process Group, ...
What the REST endpoint current does NOT offer is a way to query for the entities which's tags have been affected using said endpoint. For full coverage that would mean to iterate of 100k+ entities and check whether the list of tags is non-empty.
In other words, that resource is among the few that doesn't represent static "configuration settings". It offers to dynamically add or remove tags from entities.
